ml4bio/ml4bio.py

```diff
@@ -2149,15 +2168,14 @@ def initUI(self):
         self.models_table.verticalHeader().hide()
         self.models_table.setSelectionBehavior(QTableWidget.SelectRows)
         self.models_table.setColumnCount(8)
-        self.models_table.setHorizontalHeaderLabels(['Name', 'Type', 'Accuracy', 'Precision', 'Recall', 'F1', 'AUROC', 'AUPRC'])
-        self.models_table.setColumnWidth(0, 140)
-        self.models_table.setColumnWidth(1, 110)
-        self.models_table.setColumnWidth(2, 70)
-        self.models_table.setColumnWidth(3, 70)
-        self.models_table.setColumnWidth(4, 65)
-        self.models_table.setColumnWidth(5, 65)
-        self.models_table.setColumnWidth(6, 65)
-        self.models_table.setColumnWidth(7, 65)
+
+        #Set column names and widths
+        headerList = ['Name', 'Type', 'Accuracy', 'Precision', 'Recall', 'F1', 'AUROC', 'AUPRC']
+        fm = QFontMetrics(self.font)
+        self.models_table.setHorizontalHeaderLabels(headerList)
+        self.models_table.setSizeAdjustPolicy(QtWidgets.QAbstractScrollArea.AdjustToContents)
+        for i in range(len(headerList)):
```

Comment:
The new version of the table leaves whitespace to the right of the table if the GUI is wider than needed.  Do we all agree that is better than scaling the columns proportionally to fill the full width?  I'm happy keeping it as it is now.

Reply:
I am happy with the current layout. The columns can actually adjust their width according to text length, which is quite good.